src/App.css

```diff
@@ -36,3 +32,79 @@
     transform: rotate(360deg);
   }
 }
+
```

Comment:
![image](https://user-images.githubusercontent.com/22796757/199037705-dffab3d9-a70e-4df7-a15a-34cfaa7f13de.png)
minor but as it stands you are losing some info from the cards, its a css thing nothing to worry about really : )